pkg/device/device-util.go

```diff
@@ -287,7 +302,8 @@ func wipeFsPartition(disk string, partNum uint32) error {
 	return err
 }
 
-// GetVolumeDevPath Todo
+// GetVolumeDevPath returns the path to the volume.
+// eg: /dev/sda1, /dev/nvme0n1p1
 func GetVolumeDevPath(vol *apis.DeviceVolume) (string, error) {
 	diskMetaName := vol.Spec.DevName
 	partitionName := vol.Name[4:]
```

Comment:
can we make this as a constant

Reply:
Will change to `strings.TrimPrefix(vol.Name, "pvc-")`, so we know what is actually happening.

Reply:
Done.